### ISSUE TITLE
Add support of MDN database

### ISSUE THREAD

**Issue by socketpair** [browserslist/browserslist#522]
I mean:
https://github.com/mdn/browser-compat-data

Another example:
https://github.com/webhintio/hint

I want browserlists to be able to query browsers by expressions like `supports mdn-javascript_builtins_promise_allsettled`.

Unfortunately, current DB does not support such detailed requests.

**Comment by ai** [comment]
Do we have this file in `caniuse-db` npm package?

**Comment by socketpair** [comment]
@ai No, I did not find such filename in `caniuse-db`
```
$ find | fgrep -i promi
./features-json/promise-finally.json
./features-json/promises.json
```

**Comment by socketpair** [comment]
https://caniuse.com/#feat=mdn-javascript_builtins_promise_allsettled

original caniuse link

**Comment by ai** [comment]
Can you create an issue in Can I Use to add this data. I can't do anything without the data source (and losing data by HTTP is not an option for me because of the security risks).

**Comment by socketpair** [comment]
Yes I can... but....there are plenty of other features. Seems caniuse guys simply just do not add detailed information.

https://caniuse.com/#feat=mdn-javascript_builtins_object_fromentries

Is another feature I want to use.

That's why I'm asking you to add support of the MDN database.

**Comment by ai** [comment]
> That's why I'm asking you to support the MDN database.

How do you see this support implementation?

**Comment by socketpair** [comment]
Ummm. Instead of adding, they just refer to MDN database....I think browserslist must support this DB as well.

**Comment by socketpair** [comment]
@ai
Browserslist query language should match features from this DB. MDN db has an API.

**Comment by ai** [comment]
To support any database we need an npm package with data from that database. This npm package should be auto-updated to be sure that the data is always actual.

Do you want to create a npm package with MDN data or try to find any official package?

**Comment by socketpair** [comment]
Well. It's official package...

https://caniuse.com/#feat=mdn-javascript_builtins_object_fromentries

![image](https://user-images.githubusercontent.com/1157412/90447562-00069300-e0fd-11ea-84c4-5a410eaa37c7.png)

(click on the place like one in the picture)


This is an example of description method I want:

https://github.com/mdn/browser-compat-data/blob/master/javascript/builtins/Object.json#L600

You should not use these files directly of course. There is an API.

**Comment by ai** [comment]
> It's official package

You show a screenshot from website, but it is not [a npm package](https://www.npmjs.com/).

Browserslist can't load data from websites. Browserslist users Can I Use data because they publish this data in [caniuse-db npm package](https://www.npmjs.com/package/caniuse-db).

**Comment by socketpair** [comment]
https://www.npmjs.com/package/mdn-browser-compat-data

**Comment by ai** [comment]
Great. Can you send PR?

**Comment by socketpair** [comment]
Sorry, no. It's too hard for me to integrate all these things.

**Comment by socketpair** [comment]
https://github.com/mdn/browser-compat-data/issues/4106

**Comment by chaffeqa** [comment]
this integration woudl be 🔥 !

like socketpair, i wish i had time to help here :(

**Comment by karlhorky** [comment]
@jonathantneal @antross did either of you get further with your solution approaches from #379 ?
